custom_components/storj/api.py

```diff
@@ -72,13 +75,11 @@ async def install_uplink(self) -> bool:
 
         return True
 
-    async def authenticate(self, access_grant: str) -> bool:
+    async def authenticate(self) -> bool:
         """Test if we can authenticate with the host."""
-        self._access = self._uplink.parse_access(access_grant)
-        self._project = self._access
```

Comment:
Moving this to the constructor is the important part, along with properly setting `_project`.